### PR TITLE
docs: add portlet administration information

### PR DESCRIPTION
**Pull request by ChristianMurphy**
Markdown-ified version of the Administration/Testing guide created by the CCC team.

### CODE REVIEW

File: docs/ADMINISTRATION.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Administering the Survey Portlet
+
+Use the following steps to create an survey that you can add to your portal:
```
Comment (jgribonvald) [inline docs/ADMINISTRATION.md]: there is a typo ?  'an survey'

**Review by jgribonvald — COMMENTED** [review]
Thanks Christian !

File: docs/ADMINISTRATION.md (hunk 1 above)
Comment (ChristianMurphy) [reply docs/ADMINISTRATION.md]: Yep, that's I typo, I'll get that corrected.
Thanks @jgribonvald! :bow:

**Pull request merged**